.vsts-ci.yml

```diff
@@ -92,11 +92,11 @@ jobs:
     
 - template: build/ci/.azure-devops-package-skia.yml
   parameters:
-    poolName: '$(windowsScaledPool)'
+    vmImage: '$(linuxVMImage)'
 
 - template: build/ci/.azure-devops-package.yml
   parameters:
-    poolName: '$(windowsScaledPool)'
+    vmImage: '$(linuxVMImage)'
 
 - template: build/ci/.azure-devops-project-template-tests.yml
   parameters:
```

Comment:
Is this run on Linux? there is a `vmImageLinux: '$(linuxVMImage)'` two lines below, but there are `\` separators used, e.g:

https://github.com/unoplatform/uno/blob/83ad909bf1d5873ff07c4b78c3112e3f0e367e3e/build/ci/.azure-devops-project-template-tests.yml#L30

I don't understand how that works.

In general, I think we should stick with `/` whether we are on Linux or Windows since it should work on both.

Reply:
Jobs for this template are running on both, but this particular script is on windows. It does not always work on both unfortunately, it's a case-by-case change even if this one may.